frontend/src/metabase/visualizations/echarts/cartesian/option/axis.ts

```diff
@@ -342,77 +204,42 @@ const buildMetricAxis = (
     },
     position,
     axisLabel: {
+      margin: CHART_STYLE.axisTicksMarginY,
       show: !!settings["graph.y_axis.axis_enabled"],
       ...getTicksDefaultOption(renderingContext),
       // @ts-expect-error TODO: figure out EChart types
-      formatter: value => formatter(valueGetter(value)),
+      formatter: value => axisModel.formatter(value),
     },
   };
 };
 
-export const getYAxisName = (
```

Comment:
Moved to https://github.com/metabase/metabase/pull/36879/files#diff-a925af260455ef56ff9bc97d2376ab8cc1f92bc51256d5ab0fdb4b39a9088363R348 and slightly refactored after